sdk-tests/components/mongo-statestore.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: mongo-statestore
+spec:
+  type: state.mongodb
+  version: v1
+  metadata:
+    - name: host
+      value: localhost:27017
+    - name: databaseName
+      value: local
+    - name: collectionName
+      value: testCollection
```

Comment:
For tests, it is OK to have artificial names.

Reply:
done